components/x-live-blog-post/src/LiveBlogPost.jsx

```diff
@@ -39,7 +58,21 @@ const LiveBlogPost = (props) => {
 				className={`${styles['live-blog-post__body']} n-content-body article--body`}
 				dangerouslySetInnerHTML={{ __html: bodyHTML || content }}
 			/>
-			{showShareButtons && <ShareButtons postId={id || postId} articleUrl={articleUrl} title={title} />}
+			<div className={styles['live-blog-post__bottom-controls']}>
+				{showShareButtons && <ShareButtons postId={id || postId} articleUrl={articleUrl} title={title} />}
+				{(Boolean(backToTop) || Boolean(topRef)) && (
+					// eslint-disable-next-line jsx-a11y/click-events-have-key-events
+					<a
+						{...backToTopProps}
+						aria-labelledby="Back to top"
+						{...backToTopProps}
```

Comment:
**Question:** Is this meant to be here twice?

Reply:
🤦🏾‍♂️ I'll clean that up now.